movies/movies/WebViewController.swift

```diff
@@ -0,0 +1,88 @@
+// WebViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+//
+//  WebViewController.swift
+//  movies
+//
+//  Created by angelina on 27.10.2022.
+//
+import UIKit
+import WebKit
+
+/// Экран вебвью
+final class WebViewController: UIViewController {
+    // MARK: - Private Enum
+
+    private enum Constants {
+        static let urlShemeHostPath = "https://www.themoviedb.org/movie/"
+        static let urlFragment = "#play="
+    }
+
+    // MARK: - Private Visual Components
+
+    private lazy var webView: WKWebView = {
+        let webView = WKWebView()
+        webView.uiDelegate = self
+        webView.navigationDelegate = self
+        webView.translatesAutoresizingMaskIntoConstraints = false
+        return webView
+    }()
+
+    // MARK: - Private property
+
+    private let service = Service()
+    private var filmInfo: [VideoId]?
+
+    // MARK: - Public property
+
+    var filmIndex: Int?
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        loadWebViewData()
+        setupUI()
+    }
+
+    // MARK: - Private methods
+
+    private func setupUI() {
+        view.backgroundColor = .white
+        view.addSubview(webView)
+        createConstraints()
+    }
+
+    private func loadWebViewData() {
+        guard let index = filmIndex else { return }
+        service.loadVideos(index: index) { result in
+
+            guard result.count != 0,
+                  let url =
+                  URL(string: Constants.urlShemeHostPath + "\(index)" + Constants.urlFragment + "\(result[0].key)")
```

Comment:
```suggestion
                  URL(string:")
```
меняем на  интерполяцию